pages/osx/log.md

```diff
@@ -0,0 +1,20 @@
+# log
+
+> View, export, and configure logging systems.
+> More information: <https://www.dssw.co.uk/reference/log.html>.
+
+- Stream live system logs:
+
+`log stream`
+
+- Stream logs sent to syslog for PID 235:
+
+`log stream --process {{235}}`
+
+- Show logs sent to syslog for process named 'dasd':
+
+`log show --predicate "process == '{{dasd}}'"`
+
+- Dump all logs for the past hour:
```

Comment:
Perhaps this will read better?
```suggestion
- Print all logs from the past hour:
```

Reply:
I agree with everything you said... except this one. :) Since this one saves the logs to disk, "Print" isn't the best verb. How does this work: "Export all logs to disk for the past hour"

Reply:
Yeah, that sounds good @jchilders!